mailbox.c

```diff
@@ -28,18 +28,18 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-
 #include <uk/mbox.h>
 #include <uk/arch/time.h>
 #include <lwip/sys.h>
+#include <lwipopts.h>
 
 #include <uk/essentials.h>
 
 /* Creates an empty mailbox. */
 err_t sys_mbox_new(sys_mbox_t *mbox, int size)
 {
 	if (size <= 0)
-		size = 32;
+		size = TCPIP_MBOX_SIZE;
```

Comment:
Hum... maybe we should probably print a LWIP DEBUG message when we decide to use a default value.